FreeRADIUS/freeradius/ingest/parser.yml

```diff
@@ -0,0 +1,38 @@
+name: untitled-name
+pipeline:
+  - name: parsing
+    external:
+      name: grok.match
+      properties:
+        input_field: '{{original.message}}'
+        output_field: message
+        pattern: >-
+          \(%{INT}\)\s%{DATA:event_outcome}(\s\(%{GREEDYDATA:event_reason}\))?:\s\[%{USER}\]\s\(from\sclient\s%{DATA:network_name}\sport\s%{INT:client_port}(\scli\s(%{MAC:client_mac}|%{IP:client_ip}\svia\s%{PROTOCOL}))?\)
+        custom_patterns:
+          USER: '%{USER_EMAIL:user_email}|%{HOST}|%{DATA:user_name}'
+          HOST: 'host/%{HOSTNAME:client_domain}'
+          USER_EMAIL: '%{DATA:user_name}@%{DATA:user_domain}'
+          PROTOCOL_TUNNEL: '%{DATA:network_protocol} tunnel'
+          PROTOCOL: '%{PROTOCOL_TUNNEL}|%{DATA:network_protocol}'
+
+  - name: set_ecs_fields
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          event.kind: 'event'
+          event.category: ['authentication']
+          event.type: ['info']
+          event.dataset: 'freeradius.authentication'
+          event.category: '{{ parsing.message.event_category }}'
```

Comment:
it would be interesting to have an event with this field in test files

Reply:
I removed this fields that override the previous one (L24)